onnx_chainer/context.py

```diff
@@ -26,15 +26,31 @@ def __init__(self, model):
     def get_name(self, variable):
         str_id = id(variable)
         if str_id in self.name_list:
-            return self.name_list[str_id]
+            return self.name_list[str_id][0]
         else:
             new_name = 'v{}'.format(len(self.name_list))
             self.set_name(variable, new_name)
             return new_name
 
-    def set_name(self, variable, name):
+    def set_name(self, variable, name, pinned=False):
+        """Set ONNX node name
+
+        Arguments:
+            variable (var): target variable
+            name (str): name to be exported as ONNX node name
+            pinned (bool): if don't want to change the name on subsequent
```

Comment:
I feel these sentences are a bit redundant. How about "if True, the name will not be overwritten in subsequence process"?